index.html

```diff
@@ -11,7 +11,7 @@
 
     <header id="header" >
       <img id="header-img"src="https://res.cloudinary.com/ds5gvclyz/image/upload/v1647827635/chiffon_f83wan.jpg" width="50" height="50"
```

Comment:
The image should be wrapped with an anchor tag to make your website more profesional